src/status_im/ui/components/context_menu.cljs

```diff
@@ -1,55 +0,0 @@
-(ns status-im.ui.components.context-menu
```

Comment:
my heart is bleeding, why it's so hard to watch, how someone removes code you so diligently wrote

Reply:
Don't cry! Ask @goranjovic , you get use to it :D

Reply:
Removing code is my favorite thing

Reply:
Haha, true. Hey, I like deleting code too, just not when it's barely one month old.